notebook/static/notebook/js/notebook.js

```diff
@@ -771,27 +750,23 @@ define(function (require) {
                 if (this.mode !== 'command') {
                     this.command_mode();
                 }
+                this.get_cell(sindex).unselect();
```

Comment:
This is the logic that was there before the multiselect logic was added.  I simply reverted the multiselect code in this area.